### DISCUSSION TITLE
Dataset para proyecto final

### DISCUSSION THREAD

**Discussion by moralesluis26** [rramosp/ai4eng.v1#149] [00 - General]
Saludos profesor y monitor.
Si mi dataset no contiene la cantidad de columnas indicadas,  ¿podría completar con alguna operación o combinación de alguna de las columnas anteriores? y de poderse esto, ¿hasta cuantas columnas se pueden fabricar de esa forma?, gracias de antemano.

**Comment by rramosp** [comment]
puedes crear columnas combinando algunas existentes (multiplicando,
sumando, etc.) al menos el 60% de las columnas han de ser originales

On Tue, Jun 28, 2022 at 11:45 PM Luis David M ***@***.***>
wrote:

> Saludos profesor y monitor.
> Si mi dataset no contiene la cantidad de columnas indicadas, ¿podría
> completar con alguna operación o combinación de alguna de las columnas
> anteriores? y de poderse esto, ¿hasta cuantas columnas se pueden fabricar
> de esa forma?, gracias de antemano.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/149>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKEMMWPKJKFHP2X7BLVRPIGBANCNFSM52EKWO3A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
